src/swift/codeGeneration.js

```diff
@@ -376,6 +384,29 @@ export function structDeclarationForSelectionSet(
   });
 }
 
+function operationId(generator,  { operationName, fragmentsReferenced, source }, fragments) {
```

Comment:
I think this could be moved to `compiler.js` and put in the IR file, so that Android can have access to the same id.